src/modules/practice-client-intakes/handlers/checkout-session.ts

```
@@ -0,0 +1,48 @@
+import { getLogger } from '@logtape/logtape';
+import type Stripe from 'stripe';
+import { findPracticeClientIntakeByCheckoutSession } from '@/modules/practice-client-intakes/handlers/helpers';
+import { practiceClientIntakesRepository } from '@/modules/practice-client-intakes/database/queries/practice-client-intakes.repository';
```

ESLint

[error] `@/modules/practice-client-intakes/database/queries/practice-client-intakes.repository` import should occur before import of `@/modules/practice-client-intakes/handlers/helpers`

(import/order)